import-service/src/lambdas/importFileParser/importFileParser.lambda.ts

```diff
@@ -0,0 +1,57 @@
+import { S3Event } from 'aws-lambda';
+import S3 from "aws-sdk/clients/s3";
+
+import { parseS3CSVObjectStream } from '../../../shared/utils/s3Utils'
+
+const s3 = new S3({
+    region: process?.env?.PRODUCT_AWS_REGION || 'eu-north-1',
+})
+
+export const handler = async (event: S3Event) => {
+    console.log('Incoming request event: ', event);
+
+    try {
+        const triggerBucketName = event.Records[0].s3.bucket.name
+        const triggerBucketObjectKey = event.Records[0].s3.object.key
+
+        // Get file name
+        const fileName = triggerBucketObjectKey.split('/').reverse()[0]
```

Comment:
`fileName` and `triggerBucketObjectKey` is absolutely the same (if they are different there might be a mistake), one should handle key that is received with s3 event.